lib/src/robot/client.dart

```diff
@@ -26,6 +27,9 @@ class RobotClientOptions {
   /// The frequency (in seconds) at which to attempt to reconnect a disconnected robot. 0 (zero) signifies no reconnection attempts
   final attemptReconnectInterval = 1;
 
+  /// Whether sessions are enabled
+  final enableSessions = true;
```

Comment:
(q) Is this customizable at all? It looks like this is always on by default. Are there cases where we want to support _not_ having sessions? If not, would it be better to just have it always-on and not customizable?

Reply:
Great question. It is customizable, in the [docs for session](https://pkg.go.dev/go.viam.com/rdk/session) it is described as:

> Once implemented in an SDK, session management is opt-out from the client usage perspective.

Typescript and Python handle this similarly.

Reply:
Got it, thanks!